src/pages/home/HeaderView.js

```diff
@@ -166,7 +169,7 @@ function HeaderView(props) {
         ),
     );
 
-    const canJoinOrLeave = isChatThread || isUserCreatedPolicyRoom || canLeaveRoom;
+    const canJoinOrLeave = isChatThread || !isSelfDM || isUserCreatedPolicyRoom || canLeaveRoom;
```

Comment:
Looks like this condition caused regression https://github.com/Expensify/App/issues/37709
@ishpaul777 could you please take a look